Multiset/Multiset.playground/Sources/Multiset.swift

```diff
@@ -7,18 +7,24 @@
 
 import Foundation
 
-public struct Multiset<Element: Hashable> {
-  private var storage: [Element: UInt] = [:]
+public struct Multiset<T: Hashable> {
+  private var storage: [T: UInt] = [:]
```

Comment:
Changing the type parameter name was necessary, because Xcode 9 was failing to compile the file otherwise with this change. My assumption is that it was struggling with L16, which otherwise read:

    public init<C: Collection>(_ collection: C) where C.Element == Element {

Regardless of Xcode's problems, this change also makes that line easier to read.